app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1599,6 +1604,19 @@ public void onClick(View view) {
                 }
             });
         }
+        if (appBarLayout.getVisibility() == View.GONE) {
+            snackBarParams.setMargins(0, 0, 0, 0);
+        } else {
+            snackBarParams.setMargins(0, TOOLBAR_HEIGHT_IN_PX, 0, 0);
+        }
         snackbar.show();
     }
+
+    private static int TOOLBAR_HEIGHT_IN_PX;
```

Comment:
Per Android/Java code style guidelines, only constants (variables that are static & final) are written in all caps. I'd suggest either making this a member variable (removing `static`) or adding `final` to make it a constant.